### PR TITLE
Refactoring for more selion grid servlet html

### PR DESCRIPTION
**Pull request by mach6**
Important note: This PR builds on #202 ..

### CODE REVIEW

**Comment by sebady** [comment]
Didn't look at all the js here but the rest of the PR looks good to merge

File: server/src/main/resources/com/paypal/selion/js/listAllNodes.js [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+
+      // add a <li> item to display
+      function createInfoLi(label, item, valueType, isSelionItem) {
+        // sanitize known values which mean the proxy did not provide any data.
+        function supported(something) {
+          if (proxyClass === 'SeLionRemoteProxy') {
+            return something;
+          }
+          try {
+            if ((something === undefined) || (something === null) || (something === -1)) {
```
Comment (sundaramrajendran) [inline server/src/main/resources/com/paypal/selion/js/listAllNodes.js]: We can update this as if (!something || (something === -1)). if (something) will validate false, 0, "", NaN, null, 
undefined.

File: server/src/main/resources/com/paypal/selion/js/listAllNodes.js [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+              return 'not supported';
+            }
+          } catch (err) {
+            return 'not supported';
+          }
+          return something;
```
Comment (sundaramrajendran) [inline server/src/main/resources/com/paypal/selion/js/listAllNodes.js]: We can avoid try block as below:

if (something && (something !== -1)) {
  return something;
}
return 'not supported';

**Comment by sundaramrajendran** [comment]
Overall looks good to me

File: server/src/main/resources/com/paypal/selion/js/listAllNodes.js (hunk 2 above)
Comment (mach6) [inline server/src/main/resources/com/paypal/selion/js/listAllNodes.js]: Thx. I'll make this change.

**Pull request merged**